target/linux/qualcommax/files/arch/arm64/boot/dts/qcom/ipq6018-fap650.dts

```diff
@@ -0,0 +1,458 @@
+// SPDX-License-Identifier: MIT, GPL-2.0 or later
+/* Copyright (c) 2023, Ruslan Isaev <legale.legale@gmail.com> */
+
+/dts-v1/;
+
+
+#include "ipq6018.dtsi"
+#include "ipq6018-cp-cpu.dtsi"
+#include "ipq6018-ess.dtsi"
+
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/gpio/gpio.h>
+
+/ {
+	model = "Yuncore FAP650";
+	compatible = "yuncore,fap650", "qcom,ipq6018-cp03", "qcom,ipq6018";
+	
+	/delete-node/ rpm-glink;
+	
+	aliases {
+		ethernet0 = &dp5;
+		ethernet1 = &dp4;
+		ethernet2 = &dp3;
+		ethernet3 = &dp2;
+		ethernet4 = &dp1;
+		label-mac-device = &dp5;
+		serial0 = &blsp1_uart3;
+		led-boot = &led_system;
+		led-failsafe = &led_system;
+		led-running = &led_system;
+		led-upgrade = &led_system;
+	};
+
+	chosen {
+		stdout-path = "serial0:115200n8";
+		bootargs-append = " root=/dev/ubiblock0_1 earlycon debug iomem=relaxed";
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		reset {
+			label = "reset";
+			gpios = <&tlmm 19 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_system: led_system {
+			label = "system";
+			color = "red";
+			gpios = <&tlmm 32 GPIO_ACTIVE_HIGH>;
+		};
+
+		led_wlan2g {
+			label = "wlan2g";
+			color = "green";
+			gpios = <&tlmm 37 GPIO_ACTIVE_HIGH>;
+		};
+
+		led_wlan5g {
+			label = "wlan5g";
+			color = "blue";
+			gpios = <&tlmm 35 GPIO_ACTIVE_HIGH>;
+		};
+	};
+
+	reserved-memory {
```

Comment:
Any reason you are overriding the whole default reserved-memory?

Reply:
to avoid mistakes when something from here, something from there and something from a patches.

Reply:
That is not valid explanation, what is the RAM size on this board?

Reply:
512MB

Reply:
Then instead of this copy/paste mess overriding stuff add a 512 MB DTSI like we have for ipq807x

Reply:
done